test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -132,22 +129,130 @@ class CoronavirusLocalRestrictionsTest < ActionDispatch::IntegrationTest
       then_i_enter_a_valid_english_postcode_with_a_future_level_two_restriction
       then_i_click_on_find
       then_i_see_the_results_page_for_level_one_with_changing_restriction_levels
-      then_i_see_details_of_christmas_rules
     end
 
     it "displays restrictions changing from level two to level three" do
       given_i_am_on_the_local_restrictions_page
       then_i_enter_a_valid_english_postcode_with_a_future_level_three_restriction
       then_i_click_on_find
       then_i_see_the_results_page_for_level_two_with_changing_restriction_levels
-      then_i_see_details_of_christmas_rules
     end
 
     it "displays restrictions changing from level three to level four" do
       given_i_am_on_the_local_restrictions_page
       then_i_enter_a_valid_english_postcode_with_a_future_level_four_restriction
       then_i_click_on_find
       then_i_see_the_results_page_for_level_three_with_changing_restriction_levels
+    end
+  end
+
+  describe "before christmas" do
+    before do
+      travel_to Time.zone.parse("2020-12-25")
```

Comment:
I'd have thought this is exactly Christmas day rather than being before Christmas? as the description suggests?

Reply:
It is. I suppose I was being lazy and not calling it "on or before christmas". I'll update the test name though.

Reply:
Ah sure, sorry I thought it was more a logical flaw than a moment of laziness.